Server/controllers/participationTut.controller.js

```diff
@@ -211,3 +226,71 @@ exports.findUserPart = (result, callback) => {
       return callback("Error retrieving data", null);
     });
 };
+exports.findContentDevSolved = (req,res)=>{
+  Participation.find({username:req.params.username})
+  .then((participation)=>{
+    let arr=[]
+    let sliced=req.params.username.substr(7)
+    Participation.findOneAndUpdate({username:req.params.username,courseId:"IARE_PY",practiceSolved:{$exists:false}},    
```

Comment:
https://www.mongodb.com/docs/manual/reference/operator/query/in/

See if below is possible,

```suggestion
    Participation.findOneAndUpdate({username:req.params.username,courseId: { $in: ["IARE_PY", "IARE_C", "IARE_CPP", "IARE_JAVA"]},practiceSolved:{$exists:false}},    
```